pkg/scheduler/algorithm/priorities/resource_allocation.go

```diff
@@ -91,6 +91,8 @@ func (r *ResourceAllocationPriority) PriorityMap(
 	}, nil
 }
 
+// returns the total non-zero requests. If Overhead is defined for the pod and the PodOverhead feature
```

Comment:
comments should start with the name of function, struct, etc. Please start the comment with `getNonZeroRequests`.